usaspending_api/settings.py

```diff
@@ -845,3 +845,23 @@
     "original_loan_subsidy_cost": "original_loan_subsidy_cost",
     "business_funds_indicator": "business_funds_indicator"
 }
+
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
```

Comment:
@catherinedevlin quick question...are we using the recipients and summary awards caches (want to make sure I understand how this is working...I see the explicit code for using the location cache).

if we haven't implemented the recipients/awards caching yet, is it worth putting a note somewhere to our future selves that this is a to do when we return to this story?